### ISSUE TITLE
How to visualize the depth file?

### ISSUE THREAD

**Issue by TongWeiDP** [xy-guo/MVSNet_pytorch#20]
The depth files,such as depth_map_0002.pfm in the file of DTU training,how to visualize it？What software is used to achieve it? or using the code to visualize it ?

**Comment by XYZ-qiyh** [comment]
You can use the [python script](https://github.com/YoYo000/MVSNet/blob/master/mvsnet/visualize.py) `visualize.py` provided by original MVSNet.
Also you can use https://github.com/Todd-Qi/MVSNet-PyTorch/blob/master/tools/gray2color.py

**Comment by TongWeiDP** [comment]
> You can use the [python script](https://github.com/YoYo000/MVSNet/blob/master/mvsnet/visualize.py) `visualize.py` provided by original MVSNet.
> Also you can use https://github.com/Todd-Qi/MVSNet-PyTorch/blob/master/tools/gray2c

> You can use the [python script](https://github.com/YoYo000/MVSNet/blob/master/mvsnet/visualize.py) `visualize.py` provided by original MVSNet.
> Also you can use https://github.com/Todd-Qi/MVSNet-PyTorch/blob/master/tools/gray2color.py

Hello,I encountered a problem that the loss is very high when I first started training without the pretrained model，the batch_size is 1,num_gpus is 1,what's the reason about it? my training parameters is same as the your code. Waiting for your reply.Thanks.

**Comment by TongWeiDP** [comment]
> Hello,I trained the UCSNET network which is based on the Mvsnet network,but I encountered a problem that the loss is very high when I first started training without the pretrained model,the batch_size is one,num_gpus is one. I cannot get the author's reply,I am eager to find the the reason about it.my training parameters is same as the your code. Have you  trained the UCSNET network before? Waiting for your reply.Thanks.

**Comment by TongWeiDP** [comment]
> You can use the [python script](https://github.com/YoYo000/MVSNet/blob/master/mvsnet/visualize.py) `visualize.py` provided by original MVSNet.
> Also you can use https://github.com/Todd-Qi/MVSNet-PyTorch/blob/master/tools/gray2color.py

Hello,I trained the UCSNET network which is based on the Mvsnet network,but I encountered a problem that the loss is very high when I first started training without the pretrained model,the batch_size is one,num_gpus is one. I cannot get the author's reply,I am eager to find the the reason about it.my training parameters is same as the author's code. Have you trained the UCSNET network before? Waiting for your reply.Thanks.

**Comment by XQQ1072448274** [comment]
您好，想问下可视化深度图文件时，采用这个visualize.py时，我仅仅修改了parser.add_argument('depth_path',default='./depth_est/')
但是我并没有得到看到生成的深度图文件

**Comment by kk6398** [comment]
> 

你好，我也是遇到了相同的问题，请问你找到解决的办法了吗？

**Comment by TongWeiDP** [comment]
mvsnet的tensorflow代码里，有可视化深度图的代码



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年10月29日 20:32 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [xy-guo/MVSNet_pytorch] How to visualize the depth file? (#20) |

你好，我也是遇到了相同的问题，请问你找到解决的办法了吗？
